my-app/src/components/Move.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Move = ({history, setStepNumber, setIsXNext}) => {
+    const jumpTo = (step) => {
+    
+        setStepNumber(step);
+        setIsXNext((step % 2) === 0);
+    
+    }
+    const moves = history.map((item, index) => (
```

Comment:
Why do you even need to assign to variable? below implementation is totally fine:

```js
<ol>
  {history.map((item, index) => (
        <li>
             <button onClick={() => jumpTo(index)}>
                 {index ? `Go to move # ${index}` : `Go to game start`}
             </button>
         </li>
  ))}
</ol>
```